libs/model-ad/api-description/src/paths/comparison-tools/model-overview.yaml

```diff
@@ -2,20 +2,20 @@ get:
   tags:
     - Model Overview
   summary: Get model overview for comparison tools
-  description: Returns a list of model overview objects for use in comparison tools.
+  description: Returns a paginated list of model overview objects for use in comparison tools.
   operationId: getModelOverviews
   parameters:
     - $ref: ../query/itemQuery.yaml
     - $ref: ../query/itemFilterTypeQuery.yaml
+    - $ref: ../query/pageNumberQuery.yaml
+    - $ref: ../query/pageSizeQuery.yaml
```

Comment:
The approach taken in OC and BixArena is to specify a single "*Query" object that centralizes all the query parameters for an endpoint, including page size and number.See this example: https://github.com/Sage-Bionetworks/sage-monorepo/blob/d9902a6ac12e20fe14b44dcec138f2391f00d06b/libs/bixarena/api-description/src/paths/example-prompts.yaml#L9